src/twig.hpp

```diff
@@ -6,6 +6,7 @@
 #include <numeric>
 #include <vector>
 #include <iostream>
+#include <memory>
```

Comment:
Why is this included? I don't see where it is used.

Reply:
Good catch. I included it because I was going to try to use a `unique_ptr`, but couldn't make it work. Forgot tor remove the `#include` statement. It is now gone.